### PR TITLE
Styling set-up

### PR DESCRIPTION
**Pull request by maxgerber**
+ Add custom stylings in tailwind
 + The spacing fits within Tailwind's pre-existing scale
 + Pre-loading extensive fontset for Ubuntu and following guidelines [here](https://medium.com/clio-calliope/making-google-fonts-faster-aadf3c02a36d) 

Relates #24

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,5 +1,80 @@
...
+    screens,
+    media: {
+      sm: `(min-width: ${screens.sm})`,
+      md: `(min-width: ${screens.md})`,
+      lg: `(min-width: ${screens.lg})`,
+      xl: `(min-width: ${screens.xl})`,
+    },
+    extend: {
+      spacing: {
+        "2d5": "0.625rem", // 10px
```
Comment (ivanmauricio) [inline tailwind.config.js]: @maxgerber I think here we can use dots, i.e. `"2.5": "0.625rem"`.

**Comment by ivanmauricio** [comment]
Also good to know about font loading best practices 🤓

File: tailwind.config.js (hunk 1 above)
Comment (maxgerber) [reply tailwind.config.js]: Only thing is, is it weird to put a ```.``` within a class name considering that's how they're selected in CSS?

Comment (ivanmauricio) [reply tailwind.config.js]: So I think what happens with this is that the class is written to css as - 
```css
.mx-2\.5 {
  background-color: red;
}
```
But can be used in `html` as `className="mx-2.5"`

I know what you mean about it feeling strange though so happy to go with your decision.

**Pull request merged**